src/app/pages/login/login.page.ts

```diff
@@ -0,0 +1,82 @@
+import {
+  AfterViewInit,
+  Component,
+  ElementRef,
+  OnDestroy,
+  ViewChild,
+} from '@angular/core';
+import {
+  FormBuilder,
+  FormControl,
+  FormGroup,
+  Validators,
+} from '@angular/forms';
+import { Router } from '@angular/router';
+import { User } from '@interfaces/user';
+import { fromEvent, Subscription } from 'rxjs';
+import { LoginUser } from './../../shared/interfaces/user';
+import { AuthService } from './../../shared/services/auth/auth.service';
+
+@Component({
+  selector: 'app-login',
+  templateUrl: './login.page.html',
+  styleUrls: ['./login.page.scss'],
+})
+export class LoginComponent implements AfterViewInit, OnDestroy {
+  exampleLogin: LoginUser = {
+    email: 'admin',
+    password: 'admin',
+  };
+
+  exampleUser: User = {
+    id: 'user1',
+    firstName: 'Grzegorz',
+    lastName: 'Brzęczyszczykiewicz',
+  };
+
+  @ViewChild('submit', { static: false }) submit: ElementRef;
+  subscription: Subscription;
+  loginForm: FormGroup = this.createLoginForm();
+
+  constructor(
+    private readonly fb: FormBuilder,
+    private readonly auth: AuthService,
+    private readonly router: Router,
+  ) {}
+
+  ngAfterViewInit(): void {
+    this.subscription = fromEvent(this.submit.nativeElement, 'click').subscribe(
```

Comment:
why `fromEvent` and not `submit` event from the form? with `submit` event you get handling of `enter`  key, you don't have to unsubscribe manually etc...

Reply:
Thanks for the advice, I will change it to submit